src/apis/applicant_api.py

```diff
@@ -95,8 +95,8 @@ def post(self):
 
         cv = CV(cv_content)
 
-        with CVMapper() as cv_mapper:
-            cv_mapper.insert(cv, applicant)
+        cv_mapper = CVMapper()
+        cv_mapper.insert(cv, applicant)
```

Comment:
Same as my other comment: Why did you decide against using pythons context manager? I´m just curious 😄